src/ui/public/saved_objects/__tests__/saved_objects_client.test.js

```diff
@@ -17,121 +17,103 @@
  * under the License.
  */
 
+jest.mock('ui/kfetch',
+  () => ({
+    kfetch: async (...args) => {
+      return global.kfetchStub(...args);
+    }
+  }), { virtual: true });
+
 import sinon from 'sinon';
 import expect from 'expect.js';
 import { SavedObjectsClient } from '../saved_objects_client';
 import { SavedObject } from '../saved_object';
 
+function stubKfetch() {
+  global.kfetchStub = sinon.stub();
```

Comment:
Not a huge fan of this, much prefer switching to jest mocks when moving, and not using `global`, but that would be even more changes to this module so I understand why you didn't do it.

That said, you could avoid using `global` with something like:

```js
import { kfetch as kFetchStub } from 'ui/kfetch'

jest.mock('ui/fetch', () => ({}));

beforeEach(() => {
  // jest will trash the mock automatically no real need to cleanup in afterEach()
  require('ui/fetch').kfetch = sinon.stub();
});
```

Reply:
thanks for the tip. I have been looking for a way to mock an import without global and could not think of anything. This is much better

Reply:
Trying to run those changes gives. Any ideas?

```
    Configuration error:
    
    Could not locate module ui/fetch (mapped as /Users/nathanreese/projects/kibana/src/ui/public/fetch)
    
    Please check:
    
    "moduleNameMapper": {
      "/^ui\/(.*)/": "/Users/nathanreese/projects/kibana/src/ui/public/$1"
    },
    "resolver": undefined

```

Reply:
The error was because the test was trying to mock the import of `ui/fetch` while the real path is `ui/kfetch`. Got it to work